rs/execution_environment/benches/run-all-benchmarks.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh -eu
+##
+## Top-level script to run all execution and embedder benchmarks.
+## Usage:
+##     ./rs/execution_environment/benches/run-all-benchmarks.sh
+##
+
+printf "%-12s := %s\n" \
+    "REPEAT" "${REPEAT:=9}"
+
+RUN_BENCHMARK="${0%/*}/run-benchmark.sh"
+[ -x "${RUN_BENCHMARK}" ] || (echo "Error accessing script: ${RUN_BENCHMARK}" && exit 1)
+SUMMARIZE_RESULTS="${0%/*}/summarize-results.sh"
+[ -x "${SUMMARIZE_RESULTS}" ] || (echo "Error accessing script: ${SUMMARIZE_RESULTS}" && exit 1)
+
+run() {
+    local i="${1}"
+    local name="${2}"
+    local bench="${3}"
+    local min_file="${4}"
+    local filter="${5:-}"
+
+    repeated_file="${min_file%.*}.repeated"
```

Comment:
Not sure I understand the meaning of `repeated_file`. Does this file contain the number of repetitions done? Should it be `repeated_count_file` then?